src/server.ts

```diff
@@ -324,7 +324,7 @@ export function createServer(
 
       Object.entries(ctxRef.current.subscriptions).forEach(
         ([, subscription]) => {
-          (subscription.return || noop)();
+          subscription.return?.();
```

Comment:
The former implementation resolved in `this` being undefined inside a class method.

Reply:
I don't really understand what you mean by this. Can you elaborate a bit more please?

Reply:
Nevertheless, I like the looks of this much better! Since this PR is closed as part of #33, could you please make a separate PR with these changes?